.github/workflows/diff.yml

```diff
@@ -84,7 +87,7 @@ jobs:
           # this action may not work as expected.
           comment-token: 'diff'
           script: |
-            diff=$(for file in dist/scss/**/*.scss
+            diff=$(for file in dist/scss/**/*.scss, tokens-v2-private/css/tokens/**/*.css
```

Comment:
@rezrah I decided to mess about with this to see if I could get diff comments to work for the newbies. One of my commits I was able to get it to work 😆 but only if I specified the literal filepath to the file I knew changed. Any thoughts?

Reply:
We decided to worry about this next quarter 😄